docs/source/cli/xcube_serve.rst

```diff
@@ -260,6 +262,40 @@ are used instead.
 
 .. _datasets:
 
+
+### Viewer State Persistence
+
+The xcube server endpoint ``/viewer/state`` allows for persisting viewer state
+if configured via the ``Viewer/Persistence`` setting. It requires a ``Path``
+which may be either an absolute local path or a URL or it may be relative to
+the configuration's ``base_dìr`` setting.
+xcube Server will use this location to persist xcube Viewer states.
+therefore it should refer to a writable and have sufficient space.
+A single state takes at least 2000 bytes but can also have several megabytes
+depending on the user's activities (user shapes, layers, time-series, statistics).
+
+.. code-block:: yaml
+
+    Viewer:
+      Persistence:
+        Path: user-states
```

Comment:
What is `user-states` here? I do not see any segue for this code snippet here. Maybe writing one sentence above about it would be useful.

Reply:
See text above which refers to `Path`.